install_config/configuring_authentication.adoc

```diff
@@ -1386,3 +1386,39 @@ a secret with 32 or 64 bytes.
 <3> Encrypting secret, used to encrypt sessions. Must be 16, 24, or 32
 characters long, to select AES-128, AES-192, or AES-256.
 ====
+
+[[configuring-authentication-security]]
+== Security
+
+Administrators can prevent clients from accessing the API by their `User-Agent`
+header in the new `*userAgentMatching*` configuration setting.
+
+For example, the administrator can deny clients using particular libraries or
+particular binaries. This example denies the kubernetes 1.2 client binary and
+origin 1.1.3 binary:
```

Comment:
Maybe init cap Kubernetes, and specify OpenShift Origin instead of "origin" (but not `{product-title}`, since the version number 1.1.3 would not apply to OSE).